### ISSUE TITLE
Update GridX to v1.0

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2568]
@evanhw let me know this morning that v1.0beta2 has been made available. Need to update submodule and do a gridx test cylcle.

> Hi Tony, 
> 
> Just fyi, we already tagged v1.0beta2 which shall be the version similar as what you are using :-)
> 
> https://github.com/evanhw/gridx/tree/v1.0beta2 
> 
> Thanks!
> 
> Evan, 黄 玮

**Comment by aerwin** [comment]
Tried to upgrade to Beta 2 this morning, but ran into a JS error in GridX code. We shipped M6 with a GridX commit from May 30. By process of elimination, I discovered a JS error cropped up with the first GridX commit on June 12 (a few days before June 15 beta).

I've started a separate communication with @evanhw to try and get to the bottom of the error. But, until that's resolved, upgrading to the GridX Beta 2 is going to have to be put on hold.
